docs/deprecations.rst

```diff
@@ -136,6 +136,16 @@ Specific WebP Feature Checks
 ``True`` if the WebP module is installed, until they are removed in Pillow
 12.0.0 (2025-10-15).
 
+Get Internal Pointers to Objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. deprecated:: 11.0.0
+
+``Image.core.ImageCore.id`` and ``Image.core.ImageCore.unsafe_ptrs``
+have been deprecated and will be removed in Pillow 12 (2025-10-15).
+They were used for obtaining raw pointers to ``ImageCore`` internals. To interact with
+C code, you can use ``Image.core.ImageCore.ptr``, which returns a ``Capsule`` object.
```

Comment:
```suggestion
``Image.core.ImagingCore.id`` and ``Image.core.ImagingCore.unsafe_ptrs`` have been
deprecated and will be removed in Pillow 12 (2025-10-15). They were used for obtaining
raw pointers to ``ImagingCore`` internals. To interact with C code, you can use
``Image.core.ImagingCore.ptr``, which returns a ``Capsule`` object.
```
When you say 'ImageCore'... it should be 'ImagingCore', right?
https://github.com/python-pillow/Pillow/blob/cb2a0c40f3760411423d09defada95c54564317a/src/_imaging.c#L3777-L3778